docs/examples/tls.md

```diff
@@ -0,0 +1,101 @@
+
+# TLS Configuration
+
+To configure TLS for fluentd and fluent-bit the operator needs TLS certificates
```

Comment:
To configure TLS for `fluentd` and `fluent-bit` the operator needs TLS certificates set via the `Fluentd` and `Fluentbit` Custom Resources respectively. This can be done in two ways: